charts/rancher-vsphere-cpi/questions.yaml

```diff
@@ -63,7 +63,7 @@ questions:
 
   - variable: vCenter.ipFamily
     label: Node Address IP Family
-    description: "Select the IP family of the address(es) to be assigned to the Node. The first selected family will be the Primary."
+    description: "Select the IP families of the address(es) to be assigned to the Node. The first selected family will be the Primary."
     type: enum
```

Comment:
question: does `enum` allow you to pick multiple?

Reply:
Good question I'm not sure, nor how to test it easily. The other option would appear to be list of strings but the documentation is very light here. 

https://github.com/rancher/questions-yml-examples

I've used that as a reference but even that's archived.

Reply:
So no `enum` does not allow multiple choices, `listofstring` doesn't seem to work correctly either. I've had to resort to a comma delimited string and then split to a list in helm. Could be nicer but I don't see any other option here.